webpack.common.js

```
@@ -2,12 +2,14 @@
 const path = require("path");
 const ReactRefreshWebpackPlugin = require("@pmmmwh/react-refresh-webpack-plugin");
 const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
 
 module.exports = {
     plugins: [new ReactRefreshWebpackPlugin()],
```

Biome (1.9.4)

[error] This property value named plugins is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named plugins


(lint/suspicious/noDuplicateObjectKeys)